src/coreclr/vm/qcallentrypoints.cpp

```diff
@@ -110,6 +110,7 @@ static const Entry s_QCall[] =
     DllImportEntry(MethodTable_CanCompareBitsOrUseFastGetHashCode)
     DllImportEntry(TypeHandle_CanCastTo_NoCacheLookup)
     DllImportEntry(ValueType_GetHashCodeStrategy)
+    DllImportEntry(StreamNative_HasOverriddenSlow)
```

Comment:
```suggestion
    DllImportEntry(Stream_HasOverriddenSlow)
```
Nit: I think it looks better without the Native suffix. (We are not consistent.)